packages/byods/src/base-client/type.ts

```diff
@@ -0,0 +1,107 @@
+import {LOGGER} from '../Logger/types';
+import {BYODSError} from '../Errors';
+
+export interface LoggerConfig {
+  level: LOGGER;
+}
+
+export interface BYODSConfig {
+  logger?: LoggerConfig;
+}
+
+export type BYODSErrorEmitterCallback = (
+  err: BYODSError,
+  finalError?: boolean
+) => void;
+
+/**
+ * An interface for the `CallingClient` module.
```

Comment:
Nit: I think all this documentation is for the calling sdk, can we remove it and tailor it for the current byods sdk.

Reply:
@adhmenon Updated...!